.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Program",
+      "program": "${workspaceFolder}\\src\\extension\\src\\coreTemplateStudio.ts",
+      "outFiles": [
+        "${workspaceFolder}/**/*.js"
+      ]
+    }
+  ]
+}
```

Comment:
Does anyone know what is this file for `.vscode/launch.json`?

Reply:
Pretty sure this is like a f5 experience within the root directory. Maybe for local testing or something?

Reply:
it was generated when I was trying to debug the application but overall, it doesn't affect the webts generation experience or my script so I removed it in the new commit